functions/PayrollProcessor.Functions.Tests/Features/Employees/EmployeesTriggerTests.cs

```diff
@@ -55,15 +55,17 @@ public static class TableStorageFixtures
         public static CloudTable CreateTable(string tableName) =>
             Substitute.For<CloudTable>(new Uri($"http://127.0.0.1:10002/devstoreaccount1/{tableName}"));
 
+#nullable disable
```

Comment:
There's not a good way to make the null checker happy here. Fortunately the test will fail if this code introduces a null due to a Azure Tables library update.

Reply:
Noted, we can keep an eye on this.

Reply:
With the switch to Cosmos, we only need to use Queues and not Table Storage (or we can use Cosmos' integration into Table Storage) so this hack shouldn't be necessary at that point.